components/carousel/utils/useCarousel.ts

```diff
@@ -60,6 +61,52 @@ export const useCarouselNftEvents = ({ type }: Types) => {
   }
 }
 
+const useChainEvents = (chain, type) => {
+  const { data, loading } = useGraphql({
+    queryPrefix: 'subsquid',
+    queryName: 'lastNftListByEvent',
+    variables: nftEventVariables[type],
+    clientName: chain === 'rmrk' ? 'subsquid' : chain,
```

Comment:
this utils function would help 

<img width="512" alt="image" src="https://user-images.githubusercontent.com/31397967/192282675-e9fc400f-cf6c-4b2f-b86f-8108e1145482.png">

Reply:
thanks 👍